hosting/locale/de/LC_MESSAGES/django.po

```diff
@@ -173,6 +173,11 @@ msgstr "Anmelden"
 msgid "Create VM"
 msgstr "VM erstellen"
 
+#, fuzzy
```

Comment:
This translation seems to be for "Your VM hosted in Switzerland"

Reply:
fixed in #519